src/box/box.cc

```diff
@@ -5048,13 +5120,18 @@ box_broadcast_id(void)
 {
 	char buf[1024];
 	char *w = buf;
-	w = mp_encode_map(w, 3);
+	w = mp_encode_map(w, 4);
 	w = mp_encode_str0(w, "id");
 	w = mp_encode_uint(w, instance_id);
 	w = mp_encode_str0(w, "instance_uuid");
 	w = mp_encode_uuid(w, &INSTANCE_UUID);
 	w = mp_encode_str0(w, "replicaset_uuid");
 	w = mp_encode_uuid(w, &REPLICASET_UUID);
+	w = mp_encode_str0(w, "cluster_name");
```

Comment:
Should be added to docbot.
https://www.tarantool.io/en/doc/latest/reference/reference_lua/box_events/system_events/#box-id

Reply:
Thanks for noticing. Fixed.